recipes/cdms2/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "cdms2" %}
+{% set version = "2.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/UV-CDAT/cdms/archive/v{{ version }}.tar.gz
+  sha256: 90665e6bca64af818b1ee6c6a042f53270d2cf24ba3aa6f23513bbe6b35e7d4b
+
+build:
+  number: 0
+  skip: True  # [win or py3k] 
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - libcdms
+    - cdat_info
+    - numpy x.x
+    - libdrs_f
+    - gcc
```

Comment:
If you are using `gcc` for either `Fortran` or `libgomp` you need to add `libgfortran` (or the complete set `libgcc`) at `run` time below.